tests/test_validators.py

```diff
@@ -353,6 +353,65 @@ def check_a_two(cls, v):
         Child(a='snap')
 
 
+def test_validate_parent_all():
```

Comment:
looks like this test and the one at the bottom should swap name, so this becomes `test_validate_child_all`.